doc/source/serve/monitoring.md

```diff
@@ -127,6 +127,17 @@ In addition to the standard Python logger, Serve supports custom logging. Custom
 
 For a detailed overview of logging in Ray, see [Ray Logging](configure-logging).
 
+### JSON logging format
+You can enable JSON logging format in serve log file by setting the environment variable `RAY_SERVE_ENABLE_JSON_LOGGING=1`. After setting this environment variable, the logs will be in the following format:
```

Comment:
Does this apply to all logs or just the HTTP Proxy logs? The example implies that it's just HTTP Proxy logs.

Reply:
this example is from deployment log not from http proxy. And yes it will enable all serve log files.